src/data/real/sqrt.lean

```diff
@@ -336,6 +340,33 @@ open real
 
 variables {α : Type*}
 
+lemma exists_rat_sq_btwn_rat_aux (x y : ℝ) (h : x < y) (hy : 0 < y) :
+  ∃ q : ℚ, 0 < q ∧ x < q^2 ∧ ↑q^2 < y :=
```

Comment:
I wonder if it would be any more convenient to write this down as `0 < q ∧ q^2 ∈ set.Ioo x y`.

Reply:
It should be defeq, right?

Reply:
Yes, modulo associativity of `∧` possibly.

Reply:
No, associativity already does the right thing.